### DISCUSSION TITLE
First-party plugins in standalone CLI are out of date

### DISCUSSION THREAD

**Discussion by Flo0807** [tailwindlabs/tailwindcss#14179] [General]
Hey, I am using the Phoenix web framework, which by default installs Tailwind CSS via the Tailwind standalone CLI without Node.js.

The CLI ships along with all the first-party plugins, but I noticed, according to the [`package-lock.json`](https://github.com/tailwindlabs/tailwindcss/blob/v3.4.9/standalone-cli/package-lock.json), that the plugins have not been updated for a long time.

Can we keep the plugin versions in the CLI up to date, or is there a reason why the versions have not been updated in the last few years?

**Comment by adamwathan** [comment]
Blind spot here, hah. Thanks for raising, will get this sorted out!

**Comment by thecrypticace** [comment]
Updated! https://github.com/tailwindlabs/tailwindcss/releases/tag/v3.4.10

Thanks for pointing it out!
